localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -696,22 +698,21 @@ void NDTScanMatcher::service_trigger_node(
     (*state_ptr_)["state"] = "Initializing";
   }
   res->success = true;
-  return;
 }
 
 void NDTScanMatcher::service_ndt_align(
   const tier4_localization_msgs::srv::PoseWithCovarianceStamped::Request::SharedPtr req,
   tier4_localization_msgs::srv::PoseWithCovarianceStamped::Response::SharedPtr res)
 {
   // get TF from pose_frame to map_frame
-  auto TF_pose_to_map_ptr = std::make_shared<geometry_msgs::msg::TransformStamped>();
+  auto tf_pose_to_map_ptr = std::make_shared<geometry_msgs::msg::TransformStamped>();
   tf2_listener_module_->get_transform(
-    get_clock()->now(), map_frame_, req->pose_with_covariance.header.frame_id, TF_pose_to_map_ptr);
+    get_clock()->now(), map_frame_, req->pose_with_covariance.header.frame_id, tf_pose_to_map_ptr);
 
   // transform pose_frame to map_frame
-  const auto mapTF_initial_pose_msg = transform(req->pose_with_covariance, *TF_pose_to_map_ptr);
+  const auto initial_pose_in_map_frame = transform(req->pose_with_covariance, *tf_pose_to_map_ptr);
```

Comment:
Why not this? (Was it too long?)
```suggestion
  const auto initial_pose_msg_in_map_frame = transform(req->pose_with_covariance, *tf_pose_to_map_ptr);
```

Reply:
I'm a bit concerned about the long variable names, but I certainly think your suggestion is better. I will fix it later.

Reply:
I have done at below commit.
https://github.com/autowarefoundation/autoware.universe/pull/4270/commits/3e018bc9cec1cd028f3965a03d9ac625d1b9716a